control_panel_api/settings/base.py

```diff
@@ -1,7 +1,9 @@
 import os
 
+import boto3
 
 # SECURITY WARNING: keep the secret key used in production secret!
+
```

Comment:
Probably better to keep the comment next to the setting it comments on, but it's not a very useful comment anyway, so just remove it.

Reply:
removed